core/src/avm2/class.rs

```diff
@@ -218,6 +220,26 @@ impl<'gc> Class<'gc> {
         new_class.class_init = new_class.specialized_class_init.clone();
         new_class.class_initializer_called = false;
 
+        if params.len() > 1 {
+            panic!(
+                "More than one type parameter is unsupported: {:?}",
+                self.name()
+            );
+        }
+
+        // FIXME - we should store a `Multiname` instead of a `QName`, and use the
```

Comment:
Does #9811 change this at all?

Reply:
No - that doesn't affect the internal `QName` and `Multiname` representations